ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemRootResourceDefinition.java

```diff
@@ -73,7 +73,8 @@
                     .setAllowExpression(true).build();
     static final SimpleAttributeDefinition DEFAULT_RESOURCE_ADAPTER_NAME =
             new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.DEFAULT_RESOURCE_ADAPTER_NAME, ModelType.STRING, true)
-                    .setDefaultValue(new ModelNode().set("hornetq-ra"))
+                    // FIXME this should not be hard-coded
+                    .setDefaultValue(new ModelNode().set("activemq-ra"))
```

Comment:
This shouldn't block this merge, but we need to think through whether this requires some sort of transformation. My instinct is it's not worth it, but in general a change in a default requires a transformation of some sort. In this case the transformer would send "activemq-ra" instead of undefined, thus ensuring that all servers using the profile see the same setting. Presumably activemq-ra wouldn't work on the legacy server, so the final effect would be to require the user to explicitly configure "hornetq-ra".

That's a migration hurdle.